### ISSUE TITLE
Cut and copy in proof editor not working

### ISSUE THREAD

**Issue by rzach** [Carnap/Carnap#72]
It seems ctrl-C and ctrl-X aren't working in the proof editor for me (ie they don't put the seelction on the clipboard, ctrl-X isn't deleting the selection from the textbox).

**Comment by gleachkr** [comment]
I think it might be Ctrl-Shift-X, Ctrl-Shift-V and Ctrl-Shift-C---the other key-modifier combinations risk being intercepted by the browser. Does that do it?

**Comment by rzach** [comment]
No, CTrl+Shift+X right-aligns everything and Ctrl+Shift+C opens the Firefox DOM inspector.

(This is the Fitch editor, not the Gentzen ProofJS editor)

**Comment by gleachkr** [comment]
Ah, OK, I see (sorry, thought this was the ProofJS editor). They work sporadically for me, but mostly not. It seems to be Firefox specific (tested briefly on Chromium). I'll look into it soon.

**Comment by rzach** [comment]
That's what I thought at first too but I did make sure they work in other textboxes. Context menu Cut/Copy/Paste work so it's not a life and death issue.

**Comment by gleachkr** [comment]
OK. I noticed that this only seems to affect exercises with the `resize` option set. My theory is that the keypress for cut triggers a resize, Firefox notices the change to the element being cut from befor the cut takes place, and cancels the cut to prevent you from possibly having the wrong thing on your clipboard (similarly for copy). I've changes the resize trigger from a keypress event to an input event that only fires when the content of the textarea changes, so only AFTER the cut takes place. This seems to fix the issue for me. Does it work for you?

**Comment by rzach** [comment]
It does! Amazing, thank you!
